src/assets/_project/_blocks/utils/qg-env.js

```diff
@@ -4,6 +4,8 @@ window.qg = window.qg || {};
 window.qg.swe = window.qg.swe || {};
 window.qg.cdn = window.qg.swe.isProduction === false ? 'https://beta-static.qgov.net.au' : 'https://static.qgov.net.au';
 window.qg.swe.assets = '/assets/v3.1/latest/';
+window.qg.googleKey = window.location.hostname.search(/dev|test|localhost/) !== -1 ? 'AIzaSyCKuaFIFo7YYZXHZ5zaiEZdJx0UBoyfuAE' : 'AIzaSyAqkq7IK18bsh-TUMmNR-x9v9PsptT3LMY';
+window.qg.googleRecaptchaApiKey = window.location.hostname.search(/dev|test|localhost/) !== -1 ? '6LeNGSwUAAAAAD6o-P5UTM0FNpKjYB71Kh70F-Ud' : '6LcoIywUAAAAAN-1rq22G-bP3yxl1bBq_5nHJ6s9';
```

Comment:
This is happening via 
https://servicesmadesimpler.govnet.qld.gov.au/bitbucket/projects/CDN/repos/static-qld_cloudformation/browse/deployment_swev3.yml

Reply:
This is what is causing errors because people can not test locally and changing key back and forth make the process error prone

Reply:
https://github.com/qld-gov-au/qg-web-template/blob/master/gulp/common-tasks/js.js
This takes care of the local environment

Reply:
and what about NPM where we do not use SWE3 Bamboo pipeline ?